docs/annotate.rst

```diff
@@ -53,7 +57,7 @@ Step-by-step
 
    - Click ``>`` to move it to the right column, then click ``OK``.
```

Comment:
> click ``OK``

Do you mean `Save`?

![image](https://user-images.githubusercontent.com/1644105/88656192-d511bc00-d0c7-11ea-9fe8-a249246df0a6.png)

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/ab265a5581b93048dca8fb7592ee2254041d50ac